app/utils/requests/posts/mappings.ts

```diff
@@ -0,0 +1,29 @@
+import { ResultOf } from 'gql.tada';
+
+import { ObjectType, Post } from '@/common/types';
+import { toDate } from '@/utils/helpers';
+import { mapToUser } from '@/utils/requests/innoUsers/mappings';
+import { PostFragment } from '@/utils/requests/posts/queries';
+
+export async function mapToPosts(posts: ResultOf<typeof PostFragment>[] | undefined) {
+  return posts?.map(mapToPost) ?? [];
+}
+
+export function mapToPost(postData: ResultOf<typeof PostFragment>): Post {
+  const author = postData.author;
+
+  if (!author) {
+    throw new Error('Post contained no author');
+  }
+
+  return {
+    id: postData.documentId,
+    content: postData.comment || '',
```

Comment:
should we rename this field to comment everywhere to avoid confusion?

Reply:
we can, but i would do it in a separate issue. I started renaming, there are so many places that you need to do the renaming and we would need to test it so lets do it separately